streams/ar.m3u

```diff
@@ -358,3 +358,39 @@ https://api.new.livestream.com/accounts/679322/events/3782013/live.m3u8
 https://wowza.telpin.com.ar:1935/canal3/canal3.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="" status="online",La Nación+ (720p)
 http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UCba3hpU7EFBSk817y9qZkiA/live
+#EXTINF:-1 tvg-id="" status="timeout",La Nación+ (720p)
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UCba3hpU7EFBSk817y9qZkiA/live
+#EXTINF:-1 tvg-id="DoceMisiones.ar" status="online",Doce Misiones
+https://g2.vxral-slo.transport.edge-access.net/a05/ngrp:c12_live01-100129_all/playlist.m3u8
+#EXTINF:-1 tvg-id="CronicaTV.ar" status="online",Cronica TV
+https://g5.vxral-slo.transport.edge-access.net/b16/ngrp:cronicatv_video1-100044_all/playlist.m3u8
```

Comment:
```suggestion
#EXTVLCOPT:http-user-agent;iPhone
https://g5.vxral-slo.transport.edge-access.net/b16/ngrp:cronicatv_video1-100044_all/playlist.m3u8
```

Reply:
Thank you, as always, for the edits Lane.